src/components/Superfest/SuperfestPage/InformationBox/InformationAlertBox.style.ts

```diff
@@ -0,0 +1,17 @@
+import { Box, styled } from '@mui/material';
+
+export const InformationBox = styled(Box)(({ theme }) => ({
+  width: '80%',
+  maxWidth: '960px',
+  display: 'flex',
+  marginTop: '32px',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  alignContent: 'center',
+  alignItems: 'center',
+  overflow: 'hidden',
+  border: '2px solid',
+  padding: '32px',
+  borderRadius: '24px',
+  borderColor: '#000000',
```

Comment:
Good example which combines small things I noticed multiple times in this PR:
- numbers instead of '960px'
- spacings
- black from theme